addons/l10n_fr_pdp_reports/views/res_company_views.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_company_form_pdp" model="ir.ui.view">
+        <field name="name">res.company.form.pdp</field>
+        <field name="model">res.company</field>
+        <field name="inherit_id" ref="base.view_company_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//notebook" position="inside">
+                <page string="PDP e-Reporting" name="pdp_page" invisible="country_code != 'FR'">
+                    <group>
+                        <field name="country_code" invisible="1"/>
+                        <group>
+                            <field name="l10n_fr_pdp_enabled"/>
+                            <field name="l10n_fr_pdp_max_moves" invisible="not l10n_fr_pdp_enabled"/>
+                        </group>
+                    </group>
+                </page>
+            </xpath>
+        </field>
+    </record>
+</odoo>
```

Comment:
Add a line there, you can set up you ide to put end of file line for you btw :smile: